packages/next/client/on-demand-entries-client.js

```diff
@@ -49,11 +61,36 @@ export default async ({ assetPrefix }) => {
       }
     })
   }
-  await setup()
+  setup()
 
   async function ping () {
-    if (ws.readyState === ws.OPEN) {
-      ws.send(Router.pathname)
+    // Use WebSocket if available
+    if (ws && ws.readyState === ws.OPEN) {
+      return ws.send(Router.pathname)
+    }
+    if (!showedWarning) {
+      console.warn('onDemandEntries WebSocket failed to connect, falling back to fetch based pinging..')
```

Comment:
Let's add an err.sh link for this explaining why the fallbakc happens.

Reply:
Sounds good, added.